service-bus.api.md

```diff
@@ -0,0 +1,337 @@
+  send(message: ServiceBusMessage): Promise<void>;
+  sendBatch(messageBatch: ServiceBusMessageBatch): Promise<void>;
+}
+
+// @public
+export class ServiceBusClient {
+  constructor(connectionString: string, options?: ServiceBusClientOptions);
+  constructor(
+    hostName: string,
+    tokenCredential: TokenCredential,
+    options?: ServiceBusClientOptions
+  );
+  close(): Promise<void>;
+  getDeadLetterReceiver(
+    queueName: string,
+    receiveMode: "peekLock",
+    options?: GetReceiverOptions
+  ): Receiver<ReceivedMessageWithLock>;
+  getDeadLetterReceiver(
+    queueName: string,
+    receiveMode: "receiveAndDelete",
+    options?: GetReceiverOptions
+  ): Receiver<ReceivedMessage>;
+  getDeadLetterReceiver(
+    topicName: string,
+    subscriptionName: string,
+    receiveMode: "peekLock",
+    options?: GetReceiverOptions
+  ): Receiver<ReceivedMessageWithLock>;
+  getDeadLetterReceiver(
+    topicName: string,
+    subscriptionName: string,
+    receiveMode: "receiveAndDelete",
+    options?: GetReceiverOptions
+  ): Receiver<ReceivedMessage>;
+  getReceiver(
+    queueName: string,
+    receiveMode: "peekLock",
+    options?: GetReceiverOptions
+  ): Receiver<ReceivedMessageWithLock>;
+  getReceiver(
+    queueName: string,
+    receiveMode: "receiveAndDelete",
+    options?: GetReceiverOptions
+  ): Receiver<ReceivedMessage>;
+  getReceiver(
+    topicName: string,
+    subscriptionName: string,
+    receiveMode: "peekLock",
+    options?: GetReceiverOptions
+  ): Receiver<ReceivedMessageWithLock>;
+  getReceiver(
+    topicName: string,
+    subscriptionName: string,
+    receiveMode: "receiveAndDelete",
+    options?: GetReceiverOptions
+  ): Receiver<ReceivedMessage>;
+  getSender(queueOrTopicName: string, options?: GetSenderOptions): Sender;
+  getSessionReceiver(
+    queueName: string,
+    receiveMode: "peekLock",
+    options?: GetSessionReceiverOptions
+  ): SessionReceiver<ReceivedMessageWithLock>;
+  getSessionReceiver(
+    queueName: string,
+    receiveMode: "receiveAndDelete",
+    options?: GetSessionReceiverOptions
+  ): SessionReceiver<ReceivedMessage>;
+  getSessionReceiver(
+    topicName: string,
+    subscriptionName: string,
+    receiveMode: "peekLock",
+    options?: GetSessionReceiverOptions
+  ): SessionReceiver<ReceivedMessageWithLock>;
+  getSessionReceiver(
+    topicName: string,
+    subscriptionName: string,
+    receiveMode: "receiveAndDelete",
+    options?: GetSessionReceiverOptions
+  ): SessionReceiver<ReceivedMessage>;
+  getSubscriptionRuleManager(
+    topic: string,
+    subscription: string,
+    options?: GetSubscriptionRuleManagerOptions
+  ): SubscriptionRuleManager;
+}
+
+// @public
+export interface ServiceBusClientOptions {
+  retryOptions?: RetryOptions;
+  webSocketOptions?: WebSocketOptions;
+}
+
+// @public
+export interface ServiceBusMessage {
+  body: any;
+  contentType?: string;
+  correlationId?: string | number | Buffer;
```

Comment:
Feedback from @bterlson is that this should support UInt8array as well if it supports Buffer